src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_mon calendar--month-length_31">
```

Comment:
following BEM recommendations 
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month-length--31">
```